hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -31,6 +31,7 @@ function hadoop_usage
   hadoop_add_option "--hosts filename" "list of hosts to use in worker mode"
   hadoop_add_option "--loglevel level" "set the log4j level for this command"
   hadoop_add_option "--workers" "turn on worker mode"
+  hadoop_add_option "--jvmargs jvm arguments", "append JVM options to any existing options defined in the HADOOP_OPTS environment variable. Any defined in HADOOP_CLIENT_OPTS will be append after these jvmargs"
```

Comment:
I just checked how this looks on the CLI:

```
--buildpaths                       attempt to add class files from build tree
--config dir                       Hadoop config directory
--daemon (start|status|stop)       operate on a daemon
--debug                            turn on shell script debug mode
--help                             usage information
--hostnames list[,of,host,names]   hosts to use in worker mode
--hosts filename                   list of hosts to use in worker mode
--jvmargs jvm arguments,           append JVM options to any existing options defined in the HADOOP_OPTS environment variable. Any defined in
                                   HADOOP_CLIENT_OPTS will be append after these jvmargs
--loglevel level                   set the log4j level for this command

```
It seems a "," character has made it into the output. This may be my fault as the example change I gave could have had a comma in it.

Could you just remove the comma between the arguments and then this should be good to commit.

Thanks!